docs/pages/2019_MARVEL_Psik_MaX/sections/bands.rst

```diff
@@ -14,17 +14,17 @@ calculation steps, are performed automatically by the WorkChain.
 
 The demonstration of the workchain will be performed in a Jupyter
 notebook. To run it, follow the instructions that were given for the
-querybuilder notebook in section [sec:querybuilder]. The only difference
+querybuilder notebook in section :ref:`querybuilder`. The only difference
 is that instead of selecting the notebook in the ``querybuilder``
 directory, go to ``pw/bandstructure`` instead and choose the
 ``bandstructure.ipynb`` notebook. There you will find some example
 structures that are loaded from COD, through the importer integrated
 within AiiDA. Note that the required time to calculate the bandstructure
-for these example structures ranges from 3 minutes to almost half an
+for these example structures ranges from 6 minutes to almost half an
 hour, given that the virtual machine is running on a single core with
 minimal computational power. It is not necessary to run these examples
 as it may take too long to complete. For reference, the expected output
-band structures are plotted in Fig.[fig:workchainbandstructures].
+band structures are plotted below:
 
 .. image:: /assets/2018_PRACE_MaX/bandstructures/Al_bands.png
```

Comment:
I think we decided to keep resources like images local to the specific page. So please copy this image to `docs/pages/2019_MARVEL_Psik_MaX/sections/include/images` and adapt the reference. This way if the `2018_PRACE_MAX` image or folder gets deleted, this still works (at the cost of some duplication of course). @ltalirz do you agree with this approach?

Reply:
Yes, agreed!